modules/context/repo.go

```diff
@@ -1097,3 +1113,54 @@ func (ctx *Context) IssueTemplatesErrorsFromDefaultBranch() ([]*api.IssueTemplat
 	}
 	return issueTemplates, invalidFiles
 }
+
+func ExtractIssueConfigFromYaml(configContent []byte) (api.IssueConfig, error) {
```

Comment:
Why not `(*api.IssueConfig, error)` and return `nil, err` if there's a error?

Reply:
And I don't think we need to define this function, the body is too simple.

See:

```go
		issueConfig, err := ExtractIssueConfigFromYaml(configContent)
		if err != nil {
			return defaultIssueConfig, err
		}
```

```go
		issueConfig := api.IssueConfig{}
		if err := yaml.Unmarshal(configContent, &issueConfig); err != nil {
			return defaultIssueConfig, err
		}
```

Reply:
> Why not (*api.IssueConfig, error) and return nil, err if there's a error?

Return the default config makes life a little bit easier